app/views/notification_mailer/info_requests/_info_request.text.erb

```diff
@@ -0,0 +1,12 @@
+<%= info_request.title.html_safe %>
+<%= "-" * info_request.title.html_safe.length %>
```

Comment:
Ah, yes - that's because they aren't being translated, so because they're in a text template, they're not being escaped. I'll remove the specs and the html_safe stuff to make it clearer